src/LibraryManager.Vsix/UI/Models/LibraryIdViewModel.cs

```diff
@@ -23,7 +23,15 @@ public LibraryIdViewModel(ISearchService packageSearchService, string initialSea
         protected override Task<CompletionSet> FilterCompletions(CompletionSet input)
         {
             CompletionSet result = input;
-            int atIndex = SearchText.IndexOf('@');
+            int atIndex = SearchText.LastIndexOf('@');
+
+            //For scoped packages
+            if (SearchText.StartsWith("@", StringComparison.Ordinal))
+            {
+                int indexOfFirstSlash = SearchText.IndexOf('/');
+                atIndex = indexOfFirstSlash > 0 && SearchText.LastIndexOf('@') > indexOfFirstSlash ?
```

Comment:
You already calculated `SearchText.LastIndexOf('@')` above, but it's repeated twice again here.